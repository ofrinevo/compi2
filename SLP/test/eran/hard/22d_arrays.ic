
class MainClass { 

	static void main (string[] args) { 

		Library.println("new int[-4]");
		int[] arr = new int[-4];
		Library.println("Should not print this or get here.");

	}

}
