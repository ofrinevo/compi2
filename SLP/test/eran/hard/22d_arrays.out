new int[-4]
Runtime Error: Array allocation with negative array size!
